src/App.tsx

```diff
@@ -1,27 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
+import 'bulma';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  return (
+    <div className="App content">
+      <h1 className="title">Dynamic list of Goods</h1>
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+      <button
+        type="button"
+        data-cy="all-button"
+        className="button is-info"
+        onClick={() => getAll().then(list => setGoods(list))}
+      >
+        Load all goods
+      </button>
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+      <button
+        type="button"
+        data-cy="first-five-button"
+        className="button is-info"
+        onClick={() => get5First().then(list => setGoods(list))}
+      >
+        Load 5 first goods
+      </button>
 
-    <GoodsList goods={[]} />
-  </div>
-);
+      <button
+        type="button"
+        data-cy="red-button"
+        className="button is-info"
+        onClick={() => getRedGoods().then(list => setGoods(list))}
```

Comment:
Repeatable code  `.then(list => setGoods(list))` can be moved to separate function